src/test/features/claim/routes/new-features-consent.ts

```diff
@@ -0,0 +1,91 @@
+import { expect } from 'chai'
+import * as request from 'supertest'
+import * as config from 'config'
+
+import { attachDefaultHooks } from 'test/routes/hooks'
+import 'test/routes/expectations'
+import { checkAuthorizationGuards } from 'test/features/claim/routes/checks/authorization-check'
+import { checkEligibilityGuards } from 'test/features/claim/routes/checks/eligibility-check'
+import * as draftStoreServiceMock from 'test/http-mocks/draft-store'
+import { Paths as ClaimPaths } from 'claim/paths'
+
+import { app } from 'main/app'
+
+import * as claimStoreServiceMock from 'test/http-mocks/claim-store'
+import * as idamServiceMock from 'test/http-mocks/idam'
+
+const cookieName: string = config.get<string>('session.cookieName')
+
+const pagePath: string = ClaimPaths.newFeaturesConsent.uri
+
+describe('Feature permission: Claimant permission to try new features', () => {
+  attachDefaultHooks(app)
+
+  describe('on GET', () => {
+    checkAuthorizationGuards(app, 'get', pagePath)
+    checkEligibilityGuards(app, 'get', pagePath)
+
+    it('should render new feature consent page when user role not present and everything is fine', async () => {
```

Comment:
Please add route tests mocking failure in retrieving user roles. Please add that to both GET and POST checks.